### PR TITLE
Added Google Meet Scheduler

### LINKED ISSUE #724: Script to schedule a google meet.
# Details

- Python Script to schedule a google meet, send invitation to the respective emails and set an event for the meet in the calendar too. The script will take email ids and time as input and create a google meet for these email ids at the specified time. These email ids will also receive an invite in the gmail inbox and an event will also be created in the calendar of these email ids for the meeting.

### PR DESCRIPTION
**Pull request by Tarun-Kamboj**
# Description

Added a Python Script to schedule a google meet, send invitation to the respective emails and set an event for the meet in the calendar too. The script will take email ids of the guests and time as input and create a google meet for these email ids at the specified time. These email ids will also receive an invite in the gmail inbox and an event will also be created in the calendar of these email ids for the meeting.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Google-Meet-Scheduler/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+credentials.json
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/.gitignore:1]: this file is not needed, delete it

File: Google-Meet-Scheduler/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+The script [here](script.py) allows you to schedule a [google meeting](https://meet.google.com/) with multiple guests using [python](https://www.python.org/). You just need to provide the topic, time and list of email ids of the guests for the meeting and the [script](script.py) will handle the rest of the work.
+
+## Setup
+
+#### Step 1. You'll need to download a `credentials.json` file.
+
+- Go to [https://developers.google.com/calendar/quickstart/python](https://developers.google.com/calendar/quickstart/python).
+
+- Click the `Enable the Google Calendar API` button as shown below.
+<p align="center"><img height="250px" src="images/3.png"></p>
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/README.md:14]: image not rendered in markdown output

File: Google-Meet-Scheduler/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- Click Next on each upcoming tab, **you don't need to change anything**.
+- Download the `credentials.json` file as shown below.
+<p align="center"><img height="200px" src="images/4.PNG"></p>
+
+- Move this file to the folder where your `script.py` resides.
+
+#### Step 2. Modify this part of the code as per your needs.
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/README.md:22]: you need to take these as inputs from the user, remove this step and delete the corresponding image

File: Google-Meet-Scheduler/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+<p align="center"><img src="images/2.PNG"></p>
+
+#### Step 3. Install all the dependencies listed in the [`requirements.txt`](requirements.txt)
+- Just Run `pip install -r requirements.txt`
+- If facing any issue installing any package, just google it :)
+
+#### Step 4. Now if you run the script for the first time then you'll be requested to grant some permissions.
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/README.md:29]: None of the images here got rendered in the output. It's due to messed up paths of the image

File: Google-Meet-Scheduler/script.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                 "start": {"dateTime": event_time["start"], 'timeZone': 'Asia/Kolkata'},
+                 "end": {"dateTime": event_time["end"], 'timeZone': 'Asia/Kolkata'},
+                 "attendees": attendees,
+                 "conferenceData": {"createRequest": {"requestId": f"{uuid4().hex}", "conferenceSolutionKey": {"type": "hangoutsMeet"}}},
+                 "reminders": {"useDefault": True}
+                 }
+        event = service.events().insert(calendarId="primary", sendNotifications=True, body=event, conferenceDataVersion=1).execute()
+        return event
+
+topic = 'Topic of the meeting' # replace with suitable topic
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/script.py:46]: take input from user

File: Google-Meet-Scheduler/script.py [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+time = {
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/script.py:47]: take input from user

File: Google-Meet-Scheduler/script.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    'start':'2021-03-25T12:30:00.000000', # replace these with time of meeting
+    'end':'2021-03-25T13:00:00.000000'
+}
+guests = {
```
Comment (kaustubhgupta) [inline Google-Meet-Scheduler/script.py:51]: take input from user

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Tarun-Kamboj** [comment]
@kaustubhgupta check for the requested changes now.

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Kushal997-das

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@santushtisharma10

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**